### PR TITLE
docs: Fix metrics name rendering

### PR DESCRIPTION
**Pull request by thanethomson**
The docs theme unfortunately wraps code blocks inside tables, breaking the names and making them somewhat unreadable. This removes all the backticks from the metric names and tags.

---

#### PR checklist

- [x] Tests written/updated, or no tests needed
- [x] `CHANGELOG_PENDING.md` updated, or no changelog entry needed
- [x] Updated relevant documentation (`docs/`) and code comments, or no
      documentation updates needed

### CODE REVIEW

**Comment by adizere** [comment]
The _Name_ doesn't look that bad on a normal-sized screen, but agree the _Tags_ are unreadable.

Screenshot 2022-11-11 at 11 36 40

How can you assess that the content without backticks looks better? Is there a staging version where you can see how it looks?

**Comment by thanethomson** [comment]
> How can you assess that the content without backticks looks better? Is there a staging version where you can see how it looks?

Unfortunately not. The only way to assess this is to run `make serve-docs` from the `main` branch. We don't have this `Makefile` target for the `v0.34.x` branch. If you use Node 17+, however, you have to run the following command for it to actually work:

```bash
NODE_OPTIONS=--openssl-legacy-provider make serve-docs
```

If you want to assess how this looks across all versions, from the `main` branch you can run:

```bash
make build-docs
```

This checks out each of the `main`, `v0.37.x`, `v0.34.x` and `v0.33.x` branches right now and builds each branch's docs to `/tmp/tendermint-core-docs` as static HTML pages (this is what's run in [this workflow](https://github.com/tendermint/tendermint/blob/main/.github/workflows/docs-deployment.yml)). I usually use `python3 -m http.server` to serve the docs from these folders locally, and this effectively simulates the whole docs site locally.

Without the monospaced font, the table automatically allows horizontal scrolling to cater for the overflow. With the monospaced font, it wraps all the monospaced text, trying to squash it within the table. So it's probably some CSS-related issue.

**Review by lasarojc — APPROVED** [review]

File: docs/tendermint-core/metrics.md [hunk 1]
```diff
@@ -18,52 +18,52 @@ Listen address can be changed in the config file (see
...
+| consensus_total_txs                    | Gauge     |                 | Total number of transactions committed                                                                                                     |
+| consensus_block_parts                  | Counter   | peer_id         | Number of blockparts transmitted by peer                                                                                                   |
+| consensus_latest_block_height          | Gauge     |                 | /status sync\_info number                                                                                                                  |
+| consensus_block_syncing                | Gauge     |                 | Either 0 (not block syncing) or 1 (syncing)                                                                                                |
+| consensus_state_syncing                | Gauge     |                 | Either 0 (not state syncing) or 1 (syncing)                                                                                                |
```
Comment (cmwaters) [inline docs/tendermint-core/metrics.md:41]: I need to update these. Will do in a separate PR

**Review by cmwaters — APPROVED** [review]
